File: Practice 7/Krymov/Hse/Hse.DbGenerator/Models/Event.cs
namespace Hse.DbGenerator.Models
{
    public class Event
    {
        public const int EventIdMaxLength = 7;
        public string EventId { get; set; }

        public const int NameMaxLength = 40;
        public string Name { get; set; }
        
        public string EventType { get; set; }

        public const int OlympicIdMaxLength = 7;
        public string OlympicId { get; set; }
        
        public int IsTeamEvent { get; set; }
        
        public int NumPlayersInTeam { get; set; }

        public const int ResultNotedInMaxLength = 100;
        public string ResultNotedIn { get; set; }
        
        public Olympic Olympic { get; set; }
    }
}

File: Practice 7/Kalmykov/db-filler/src/main/kotlin/com/example/databasefiller/models/event/Event.kt
package com.example.databasefiller.models.event

import com.example.databasefiller.models.Olympic

const val MAX_PLAYERS_IN_TEAM = 7
const val MIN_PLAYERS_IN_TEAM = 4

data class Event(
    var eventId: String,
    val name: String,
    val eventType: String,
    val olympic: Olympic,
    val isTeamEvent: Int,
    val playersNumberInTeam: Int,
    val resultNotedIn: String
)
